src/Selenium2Driver.php

```diff
@@ -692,6 +690,37 @@ public function setValue($xpath, $value)
         $this->executeJsOnElement($element, $script);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function sendKeys($xpath, $value)
+    {
+        $element = $this->findElement($xpath);
+        $elementName = strtolower($element->name());
+
+        if ('select' === $elementName) {
```

Comment:
Why this IF is needed. Instead I'm proposing to allow only these elements in method:

* `input` HTML elements with specific `type` attribute values (already done)
* maybe `textarea` elements